core/amber/src/main/scala/edu/uci/ics/amber/engine/common/ambermessage/WorkflowMessage.scala

```diff
@@ -1,12 +1,42 @@
 package edu.uci.ics.amber.engine.common.ambermessage
 
 import edu.uci.ics.amber.engine.common.virtualidentity.ActorVirtualIdentity
+import edu.uci.ics.amber.engine.common.virtualidentity.util.CLIENT
 
-sealed trait WorkflowMessage extends Serializable {
-  val from: ActorVirtualIdentity
-  val sequenceNumber: Long
+import scala.util.hashing.MurmurHash3
+
+object ChannelEndpointID {
+  def apply(endpointWorker: ActorVirtualIdentity, isControlChannel: Boolean): ChannelEndpointID = {
+    new ChannelEndpointID(endpointWorker, isControlChannel)
+  }
+}
```

Comment:
The constructor seems exactly the same as the original constructor. is this `apply` really needed?